phpseclib/Crypt/Base.php

```diff
@@ -2001,11 +2057,48 @@ function _createInlineCryptFunction($cipher_code)
      * for which $mode the lambda function was created.
      *
      * @access private
-     * @return &Array
+     * @return Array &$functions
      */
     function &_getLambdaFunctions()
     {
         static $functions = array();
         return $functions;
     }
+
+    /**
+     * Generates a digest from $bytes
+     *
+     * @see _setupInlineCrypt()
+     * @access private
+     * @param $bytes
+     * @return String
+     */
+    function _trapdoor($bytes)
+    {
+        if (!defined('CRYPT_BASE_WHIRLPOOL_AVAILABLE')) {
```

Comment:
I guess that actually could work since the `CRYPT_BASE_WHIRLPOOL_AVAILABLE` isn't really used outside of that function, however, I'm still lacking enthusiasm about doing a static variable in the method as it's just inconsistent with the way phpseclib does other similar tests (as noted above).

And when phpseclib does do static variables in methods it's not doing them for 

I mean I'll do it if it helps get this merged but I think it is a little inconsistent for the php4 / 1.0 branch.